packages/xo-web/src/common/form/number.js

```diff
@@ -7,46 +7,54 @@ import decorate from '../apply-decorators'
 // it provide `data-*` to add params to the `onChange`
 const Number_ = decorate([
   provideState({
+    initialState: ({ value }) => {
+      if (value == null) {
+        value = ''
+      }
+      return {
+        value: String(value),
+      }
+    },
```

Comment:
```js
initialState: ({ value }) => ({
      value: value == null ? '' : String(value)
    }),
```

Reply:
According to this https://github.com/vatesfr/xen-orchestra/pull/4690/files#diff-db976062fb720a858cabbab0c338fa4dR57 `value` must be a Number, why do you initialize it with a `string` ?

Reply:
There have two values: value provided by the parent which is a `number` and the value of the state which is a string.

This PR don't change the old behavior, the value passed to the input is a `String` and the value passed to the `onChange` is a number. See https://github.com/vatesfr/xen-orchestra/pull/4690/files#diff-db976062fb720a858cabbab0c338fa4dL42